core/domain/prod_validation_jobs_one_off.py

```diff
@@ -330,6 +330,16 @@ class ExplorationSnapshotMetadataModelAuditOneOffJob(
     def entity_classes_to_map_over(cls):
         return [exp_models.ExplorationSnapshotMetadataModel]
 
+    @staticmethod
+    def reduce(key, values):
+        """Yields number of fully validated models or the failure messages."""
+        if 'fully-validated' in key:
```

Comment:
Can this be made into a named constant and used everywhere it's relevant? (May need some restructuring of the messages, not sure.)

Reply:
Done.